features/step_definitions/stepdefs.js

```diff
@@ -180,6 +178,7 @@ AfterAll(async function () {
 
 Before(async function () {
     await captain.clearAllRequests();
+    await client('key_value').truncate();
```

Comment:
Moved this to execute before each scenario to ensure the db is clean for each one. This does slow things down a little though, not sure if there is a better solution 🤔

Reply:
Changed my mind about this and switched to a unit test